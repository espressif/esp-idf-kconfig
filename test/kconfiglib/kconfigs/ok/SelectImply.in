mainmenu "Test simple indirect set"


    comment "Test indirect set for bools - select"

    config I
        bool
        default y
        select J

    config J # Should be y, I is y
        bool "j"

    config K
        bool
        default y
        select L

    config L # Should be y, K is y, no matter the depends on
        bool "l"
        depends on NOT_DEFINED


    comment "Test indirect set for bools - imply"

    config M
        bool
        default y
        imply N

    config N # Should be y, M is y
        bool "n"
        default n

    config O
        bool
        default y
        imply P

    config P # Shouldn't be in the output sdkconfig; depends on is n/false
        bool "p"
        default y
        depends on NOT_DEFINED
