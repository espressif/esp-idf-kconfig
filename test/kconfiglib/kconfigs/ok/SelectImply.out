#
# Automatically generated file. DO NOT EDIT.
# Espressif IoT Development Framework (ESP-IDF)  Project Configuration
#

#
# Test indirect set for bools - select
#
# default:
CONFIG_I=y
# default:
CONFIG_J=y
# default:
CONFIG_K=y
# default:
CONFIG_L=y

#
# Test indirect set for bools - imply
#
# default:
CONFIG_M=y
# default:
CONFIG_N=y
# default:
CONFIG_O=y
